time.js

```diff
@@ -0,0 +1,50 @@
+document.addEventListener("DOMContentLoaded", function () {
+    let count = 0;
+    let timerInterval;
+    let timerDisplay = document.getElementById("timer");
+    let countList = document.getElementById("count_list");
+  
+    function startTimer() {
+      if (!timerInterval) {
+        timerInterval = setInterval(() => {
+          count++;
+          timerDisplay.textContent = count;
+        }, 1000);
+      }
+    }
+  
+    function pauseTimer() {
+      clearInterval(timerInterval);
+      timerInterval = undefined;
+    }
+  
+    function resumeTimer() {
+      if (!timerInterval) {
+        startTimer();
+      }
+    }
+  
+    function stopTimer() {
+      clearInterval(timerInterval);
+      timerInterval = undefined;
+      count = 0;
+      timerDisplay.textContent = count;
+    }
+  
+    function countTimer() {
```

Comment:
https://github.com/PUP-BSIT/exercise-12-norona-marynicole/blob/fd7c3d27c3e9e1c289a0752467a55dc8b5fac9c5/time.js#L34-L38

- you already removed the count list button and function in your html file, you should also remove this in your java script file.